### PR TITLE
Add cost center in account

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR adds the cost center information in the account.

In the account edit modal:
image

In the account registry: 
image

### CODE REVIEW

File: server/models/schema.sql [hunk 1]
```diff
@@ -18,6 +18,7 @@ CREATE TABLE `account` (
   `hidden` TINYINT(1) UNSIGNED DEFAULT 0,
   `created` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `reference_id` TINYINT(3) UNSIGNED DEFAULT NULL,
+  `cost_center_id` MEDIUMINT(8) UNSIGNED NULL,
```
Comment (jniles) [inline server/models/schema.sql:21]: You will want to put a FOREIGN KEY here.

File: test/integration/accounts.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ describe('(/accounts) Accounts', () => {
   const responseKeys = [
     'id', 'enterprise_id', 'locked', 'created', 'reference_id',
     'number', 'label', 'parent', 'type_id', 'type', 'translation_key',
+    'cost_center_id',
```
Comment (jniles) [inline test/integration/accounts.js:30]: Can you add a test where you update a cost center to the integration tests?  Just for completeness.

**Comment by mbayopanda** [comment]
@jniles changes landed

**Review by jniles — APPROVED** [review]
LGTM.  

bors r+

**Pull request merged**